net/dns/flush_default.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !windows
+// +build !windows
+
+package dns
+
+func Flush() error {
```

Comment:
forget to add a file to this PR?

Also, a convention is to put the exported "func Flush" in a portable file where you can attach docs (just once) and then have 2+ unexported versions without docs.

Reply:
There was already a flush_windows.go, called from router_windows.go.

Added unexported flushCaches(). Windows retains its public Flush() method, because router_windows cannot easily get access to the dns.Manager to call FlushCaches().